tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/GivenUsingSuite.scala

```diff
@@ -688,7 +688,7 @@ class GivenUsingSuite extends BaseDottySuite {
       )
     )
 
-    runTestAssert[Stat]("import File.{ given TC, given AC, _ }")(
+    runTestAssert[Stat]("import File.{ given TC, given AC, * }")(
```

Comment:
as we still need to support `_` it could be good to also keep a test for import + given + '_'